### ISSUE TITLE
`workspace_blacklist` not working

### ISSUE THREAD

**Issue by Tetrapak0** [vyfor/cord.nvim#58]
### Description

I want to mask my user directory when I edit a file in it, as the folder name is basically my full name.
I've tried blocking just the username or just the whole path. Neither worked.

### Operating system

Windows 10 Pro 22H2 19041

### Neovim version

```markdown
NVIM v0.10.0
Build type: Release
LuaJIT 2.1.1713484068
```

### Steps to reproduce

1. Put a directory name into your `workspace_blacklist` array.
2. Open Neovim in that folder
3. Check Discord

### Expected behavior

The workspace name should be hidden.

### Actual behavior

The workspace shows normally.

### Minimal configuration

```Lua
{
        "vyfor/cord.nvim",
        build = "./build",
        event = LazyFile,
        init = function()
            local username, homepath
            if vim.loop.os_uname().sysname == "Windows_NT" then
                username = os.getenv("USERNAME")
                homepath = os.getenv("USERPROFILE")
            else
                username = os.getenv("USER")
                homepath = os.getenv("HOME")
            end
            if username == nil then
                username = ""
            else
                username = tostring(username)
            end
            if homepath == nil then
                homepath = ""
            else
                homepath = tostring(homepath)
            end
            require("cord").setup({
                display = {
                    workspace_blacklist = {username, homepath},
                },
            })
        end,
    },
```

**Comment by vyfor** [comment]
Hey! Just to clarify, are you trying to hide any workspace located under your user directory, or just the user directory itself?

**Comment by Tetrapak0** [comment]
Just the user directory itself.

**Comment by Tetrapak0** [comment]
Updated issue to show test with full path

**Comment by vyfor** [comment]
Can confirm. The first update is sent before checking the blacklist.
/

**Comment by vyfor** [comment]
Fixed in #61, thanks! Let me know if you run into any issues.
